stream/personalization.py

```diff
@@ -0,0 +1,122 @@
+class Personalization(object):
+    def __init__(self, client, token):
+        """
+
+        :param client: the api client
+        :param token: the token
+        """
+
+        self.client = client
+        self.token = token
+
+    def get(self, url, **params):
```

Comment:
I find it very confusing to see RW functions for feeds and collections in the same namespace

Eg.

`personalization.get` for feeds
`personalization.select` for data

What about this:

```python
personalization.collections.get(*args) #then we have a nicer name than select available as well :)
personalization.collections.upsert(*args)

personalization.feeds.get(*args)
personalization.feeds.post(*args)
personalization.feeds.delete(*args)
```

Reply:
yeah but lets drop it down a level:

```
client.collections.upsert(collection_name, items)
client.collections.upsert(‘user’, [{‘id’: 123, ‘username’: ‘tschellenbach’}, {}.....])
client.collections.upsert(‘product’, [{‘id’: 123, ‘username’: ‘tschellenbach’}, {}.....])
client.collections.delete(collection_name, foreign_ids)
client.collections.delete(‘user’, [123])
client.collections.select(collection_name, foreign_ids)
client.collections.select(‘user’, [123])

```